### PR TITLE
Recipes: add iostat

### PR DESCRIPTION
**Pull request by lcpz**
A disk I/O widget using `iostat` from `sysstat` utilities. Taken from [here](https://github.com/lcpz/lain/issues/364#issuecomment-333805465).

### CODE REVIEW

File: recipes/watch.mdwn [hunk 1]
```diff
@@ -166,3 +166,23 @@ local mybattery = awful.widget.watch(
...
+    function(widget, stdout)
+        for line in stdout:match("(sd.*)\n"):gmatch("(.-)\n") do
+            local device, tps, read_s, wrtn_s, read, wrtn =
+            line:match("(%w+)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)")
+            --                  [1]  [2]     [3]     [4]   [5]
+            iotable[device] = { tps, read_s, wrtn_s, read, wrtn }
+        end
+
+        -- customize here
+        widget:set_text("sdb: "..iotable["sda"][2].."/"..iotable["sda"][3]) -- read_s/wrtn_s
```
Comment (psychon) [inline recipes/watch.mdwn]: This mixes up `sdb` and `sda` a bit, but whatever...

Comment (lcpz) [reply recipes/watch.mdwn]: Will correct it.

File: recipes/watch.mdwn [hunk 2]
```diff
@@ -62,6 +62,26 @@ local cmus, cmus_timer = awful.widget.watch(
...
+
+```lua
+-- disk I/O using iostat from sysstat utilities
+local iotable = {}
+local iostat = awful.widget.watch("iostat -dk", 2, -- in Kb, use -dm for Mb
+    function(widget, stdout)
+        for line in stdout:match("(sd.*)\n"):gmatch("(.-)\n") do
+            local device, tps, read_s, wrtn_s, read, wrtn =
+            line:match("(%w+)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)%s*(%d+,?%d*)")
+            --                  [1]  [2]     [3]     [4]   [5]
```
Comment (actionless) [inline recipes/watch.mdwn]: sorry for the late review, why not just 

```lua
iotable[device] = {
  tps=tps,
  read_s=read_s
```

?

Comment (lcpz) [reply recipes/watch.mdwn]: What for?

Comment (actionless) [reply recipes/watch.mdwn]: to avoid the comment on line 75 and write it more readable:
`iotable.sda.read_s` instead of `iotable["sda"][2]`

Comment (lcpz) [reply recipes/watch.mdwn]: OK, change it yourself if you want. I'm too lazy to open another PR for a couple of lines. 😺

**Pull request merged**